### PR TITLE
Appeals Status -> What Happens Next?

### PR DESCRIPTION
**Pull request by markgreenburg**
Element displays one or more next events for each appeal (depending on how many are available). 
Content not finalized - just a placeholder for now.

**Desktop:**
screen shot 2017-11-21 at 9 16 49 pm

**Mobile:**
screen shot 2017-11-21 at 9 17 15 pm

### CODE REVIEW

File: test/claims-status/components/appeals-v2/WhatsNext.unit.spec.jsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+describe('<WhatsNext/>', () => {
+  it('renders', () => {
+    const wrapper = shallow(<WhatsNext {...defaultProps}/>);
+    expect(wrapper.type()).to.equal('div');
+  });
+
+  it('renders a list of all next events for a given currentStatus', () => {
+    const wrapper = shallow(<WhatsNext {...defaultProps}/>);
+    const nextEventList = wrapper.find('NextEvent');
+    expect(nextEventList.length).to.equal(2);
```
Comment (cvalarida) [inline test/claims-status/components/appeals-v2/WhatsNext.unit.spec.jsx]: This is fine, but maybe `.to.equal(defaultProps.nextEvents.length)` would be more clear _why_ it's 2?

File: test/claims-status/utils/helpers.unit.spec.js [hunk 2]
```diff
@@ -469,4 +470,16 @@ describe('Disability benefits helpers: ', () => {
       expect(contents.description).to.equal('Your current appeal status is unknown at this time');
     });
   });
+
+  describe.only('getNextEvents', () => {
+    it('returns an array of next event objects', () => {
+      const type = STATUS_TYPES.nod;
+      const nextEvents = getNextEvents(type);
+      expect(nextEvents.length).to.equal(2);
```
Comment (cvalarida) [inline test/claims-status/utils/helpers.unit.spec.js]: It isn't immediately clear why this is 2. Could you put a comment in here explaining why?

File: test/claims-status/utils/helpers.unit.spec.js [hunk 3]
```diff
@@ -469,4 +470,16 @@ describe('Disability benefits helpers: ', () => {
...
+      const firstEvent = nextEvents[0];
+      const secondEvent = nextEvents[1];
+      expect(Object.keys(firstEvent).length).to.equal(4);
+      expect(Object.keys(secondEvent).length).to.equal(4);
```
Comment (cvalarida) [inline test/claims-status/utils/helpers.unit.spec.js]: Same here, maybe.

**Review by cvalarida — APPROVED** [review]
Looks good! I just had a few nitpicky things.

File: src/sass/claims-status.scss [hunk 4]
```diff
@@ -860,3 +860,47 @@ h1:focus {
...
+        display: inline-block;
+        vertical-align: middle;
+        width: 1em;
+        height: 1em;
+        content: '';
+        border: 4px solid $color-gray-medium;
+        border-radius: 50%;
+        background: $color-gray-medium;
+        margin-left: -.6em;
+      }
```
[the remark this answers is not in the document]
Comment (markgreenburg) [reply src/sass/claims-status.scss]: ahh, good eye!

File: src/js/claims-status/utils/appeals-v2-helpers.jsx [hunk 5]
```diff
@@ -193,3 +193,52 @@ export function getEventContent(event) {
...
+  switch (currentStatus) {
+    case STATUS_TYPES.nod:
+      return [
+        {
+          title: 'Additional evidence',
+          description: `VBA must reveiw any additional evidence you submit prios to certifying
+          your appeal to the Board of Veterans’ Appeals. This evidence could cause VBA
+          to grant your appeal, but if not, they will need to produce an additional
+          Statement of the Case.`,
+          cardNumber: '11 months',
```
[the remark this answers is not in the document]
Comment (markgreenburg) [reply src/js/claims-status/utils/appeals-v2-helpers.jsx]: Yeah, I was at a bit of a loss for the right name.

**Pull request merged**